Florida_Analysis_Py_File.py

```diff
@@ -0,0 +1,534 @@
+# Florida Data Analysis - Opioid Shipment And Overdosage Related Mortality
+
+## Pre-Post Analysis Of Opioid Shipments In Florida
+
+# importing the required libraries
+import pandas as pd
+import numpy as np
+import altair as alt
+import statsmodels.api as sm
+import statsmodels.formula.api as smf
+
+# loading data for drug prescriptions in Florida and neighbouring states
+prescriptions = pd.read_csv(
+    r"C:\Users\annap\OneDrive\Desktop\Opioid Project\Prescriptions.csv",
+    low_memory=False,
+)
+prescriptions.columns
+
+prescriptions.head(3)
+
+prescriptions.columns
+# reducing the number of columns in the drug prescriptions dataset by including only the relevant attributes to create a new dataset
+
+prescriptions_reduced = prescriptions[
+    [
+        "DRUG_CODE",
+        "DRUG_NAME",
+        "QUANTITY",
+        "UNIT",
+        "STRENGTH",
+        "CALC_BASE_WT_IN_GM",
+        "DOSAGE_UNIT",
+        "Product_Name",
+        "Ingredient_Name",
+        "Measure",
+        "MME_Conversion_Factor",
+        "dos_str",
+        "Year",
+        "Month",
+        "StateFIPS",
+        "StateName",
+        "CountyFIPS",
+        "state_abbrev",
+        "FIP_unique",
+        "Population",
+        "county_test",
+    ]
+]
+
+prescriptions_reduced.head(5)
+
+# creating a copy of reduced dataset of prescriptions and converting some of the attributes to appropriate data type
+
+prescriptions_reduced_copy = prescriptions_reduced.copy()
+
+prescriptions_reduced_copy["Year"] = prescriptions_reduced_copy["Year"].astype("int64")
+prescriptions_reduced_copy["DRUG_CODE"] = prescriptions_reduced_copy[
+    "DRUG_CODE"
+].astype("int64")
+prescriptions_reduced_copy["Month"] = prescriptions_reduced_copy["Month"].astype(
+    "int64"
+)
+prescriptions_reduced_copy["Population"] = prescriptions_reduced_copy[
+    "Population"
+].astype("int64")
+
+prescriptions_reduced_copy.rename(
```

Comment:
Hey Annapurna, did you rename these variables here? I can see and understand you doing this. Nice clean-up of the variable names!

Reply:
Yes, I thought it would look better and more formal, more presentable.